emgapianns/management/lib/sanity_check.py

```diff
@@ -1,33 +1,74 @@
 import glob
 import os
 from subprocess import check_output, CalledProcessError
+import mysql.connector
 
 import logging
 
 from emgapianns.management.lib.uploader_exceptions import NoAnnotationsFoundException, \
     UnexpectedLibraryStrategyException, QCNotPassedException, CoverageCheckException
-from emgapianns.management.lib.utils import read_chunkfile
+from emgapianns.management.lib import utils
 from emgapianns.management.webuploader_configs import get_downloadset_config
+from backlog import models as backlog_models
 
 logger = logging.getLogger(__name__)
+BACKLOG_CONFIG = os.environ.get('BACKLOG_CONFIG')
+
+
+def get_result_status(db_name, accession):
+    """
+    no_qc: upload qc results only. No downloads
+    no_cds: upload qc and taxonomy. Antismash tab optional if files present.
+    no_tax: standard upload (tax files are optional)
+    no_cds_tax: use no_qc config. Upload qc. Processed reads and other ncRNA downloadable
+
+    :param accession:
+    :return: result status
+    """
+    config = utils.read_config(BACKLOG_CONFIG, db_name)
+    cnx = mysql.connector.connect(**config)
+    cursor = cnx.cursor()
+    command = "SELECT `result_status` FROM `emg_backlog_2`.`AnnotationJob` " \
```

Comment:
You can improve the readability of this by using, (apologies as this is a bit pedantic):

```python
    command = (
        "SELECT result_status FROM emg_backlog_2.AnnotationJob"
        "INNER JOIN emg_backlog_2.RunAnnotationJob"
        "ON AnnotationJob.id = RunAnnotationJob.annotation_job_id"
        "INNER JOIN emg_backlog_2.Run ON "
        "Run.id = RunAnnotationJob.run_id WHERE Run.primary_accession = %s"
    )
    cursor.execute(command, (accession,))
```
there are 2 bits there:
 - https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation
 - the execute mode can get parametrs that will use on the query (https://dev.mysql.com/doc/connector-python/en/connector-python-api-mysqlcursor-execute.html).
 
The execute method is safer because it will prevent SQL injections (not very relevant here but I think it's good to use anyway)